src/OrderFormAcceptanceTests.Steps/Steps/CompleteOrder.cs

```diff
@@ -63,6 +70,7 @@ public void GivenThatTheUserIsOnTheConfirmCompleteOrderScreen()
             ThenTheConfirmCompleteOrderScreenIsDisplayed();
         }
 
+        [Given(@"the Order completed screen is displayed")]
         [Then(@"the Order completed screen is displayed")]
```

Comment:
Since this Given and Then are the same, change the keyword to StepDefinition which is generic and will map to all step types, then remove the duplicate.